api/api.go

```diff
@@ -175,8 +175,8 @@ func (b *BlockChainAPI) GetTransactionByHash(
 		Hash:             txHash,
 		BlockHash:        &rcp.BlockHash,
 		BlockNumber:      (*hexutil.Big)(rcp.BlockNumber),
-		From:             from,
-		To:               tx.To(),
+		From:             from.Hex(),
+		To:               tx.To().Hex(),
```

Comment:
I think there's a possible nil pointer dereference, because `tx.To()` returns a value of type: `*common.Address`.